examples/sinatra_example/Gemfile

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+# Specify your gem's dependencies in sinatra_example.gemspec
+gemspec
+
+gem "rake", "~> 13.0"
+
+gem "test-unit", "~> 3.0"
+
+gem "rubocop", "~> 1.7"
+
+gem "opentelemetry-instrumentation-pg"
+gem "opentelemetry-instrumentation-sinatra"
+gem "pg"
```

Comment:
should we make it run daily with some automated tests?

Reply:
Would be a good future PR yea